types/humblebee__styled-components-breakpoint/tslint.json

```
@@ -0,0 +1,8 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-outside-dependencies": false,
+        "no-single-declare-module": false,
+        "prefer-declare-function": false
+      }
```

```suggestion
    "extends": "dtslint/dt.json"
```
please add inline exclusions, it's a single file, would be better to maintain and for readability

---

I have added inline exclusions now. No outside dependencies can't be inlined afaik, since it refers to a project reference to csstype.